### PR TITLE
Create zb-client command

### LINKED ISSUE #4: Implement Zeebe Client Functions and Structs
This allows us to programatically create Zeebe Clients credentials to be able to connect to the clusters

http://docs.camunda.io/docs/reference/cloud-console-api-reference/

### LINKED ISSUE #11: Create all the structs for Camunda Cloud Payloads
https://mholt.github.io/json-to-go/

### PR DESCRIPTION
**Pull request by mcruzdev**
Creating zb-client structure for get, create, delete actions.

### CODE REVIEW

**Comment by salaboy** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **salaboy**
You can assign the PR to them by writing `/assign @salaboy` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=camunda-community-hub%2Fcamunda-cloud-go-client).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/camunda-community-hub/camunda-cloud-go-client/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by salaboy** [comment]
@mcruzdev: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | 4efac0fdf035544c7767fda7fd1d0d2223ad9199 | [link](http://dashboard-jx.34.67.22.199.nip.io/namespaces/jx/pipelineruns/pr-xbfvd) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: cmd/zbClient.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2021 Matheus Cruz <matheuscruz.dev@gmail.com>
```
Comment (salaboy) [inline cmd/zbClient.go]: @mcruzdev we don't use these headers with names like this. We leave the License only here.

**Comment by salaboy** [comment]
@mcruzdev: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | be13a5b4d6dcad6fd82580c1e5cdd1883aeeeac2 | [link](http://dashboard-jx.34.67.22.199.nip.io/namespaces/jx/pipelineruns/pr-d5h4k) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: cmd/zbClient.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		zeebeClientGetExample + zeebeClientCreateExample + zeebeClientDeleteExample,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("zb-client called")
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(zbClientCmd)
+
+	// Here you will define your flags and configuration settings.
```
Comment (salaboy) [inline cmd/zbClient.go]: @mcruzdev I will wait for this to be implemented to merge this PR, is that ok? There is no reason to merge this PR before we have some actions implemented.

Comment (mcruzdev) [reply cmd/zbClient.go]: I was thinking about this PR as a base for some issues

**Comment by salaboy** [comment]
@mcruzdev: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | f1a9058abc38f238ddb7d410be862bb8acd132fb | [link](http://dashboard-jx.34.67.22.199.nip.io/namespaces/jx/pipelineruns/pr-wjts4) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by salaboy** [comment]
@mcruzdev: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | 112424059a9d11dfcb7152576c0ef5315514a0f1 | [link](http://dashboard-jx.34.67.22.199.nip.io/namespaces/jx/pipelineruns/pr-cjth9) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: cmd/zbClient.go (hunk 2 above)
Comment (mcruzdev) [reply cmd/zbClient.go]: I implemented zb-client get --cluster action

After, I can get other issues like create, delete ...

**Pull request merged**